src/main/java/com/project/bookstore/model/User.java

```diff
@@ -0,0 +1,23 @@
+package com.project.bookstore.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Data

```

Comment:
using default lombok `@Data` annotation on entities can lead to bugs, use other required annotations except `@Data` and default `@EqualsAndHashCode` 
```suggestion
```